modules/core/test/test_allocation.cpp

```diff
@@ -0,0 +1,25 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html.
+
+#include "test_precomp.hpp"
+#include <cmath>
+
+namespace opencv_test {
+namespace allocation {
+
+TEST(Core_Allocation, alignedAllocation)
```

Comment:
Please move this test into `test_misc.cpp` and keep make it disabled (add `DISABLED_` prefix).

Reply:
I can move it, no problem. However, I believe it is a good idea to keep the test enabled. The time cost is less than a millisecond and the value is testing the core memory allocator for all of opencv. That's value..yes?

Reply:
> The time cost is less than a millisecond

OK, Agreed.

---

> memory allocator for all of opencv

From code coverage perspective it is already tested everywhere.